test/regress/regress1/sygus/issue3461.sy

```diff
@@ -1,17 +1,17 @@
 ; EXPECT: unsat
-; COMMAND-LINE: --sygus-out=status
+; COMMAND-LINE: --lang=sygus2 --sygus-out=status
 
 (set-logic ALL_SUPPORTED)
 
 (declare-datatype Doc ((D (owner Int) (body Int))))
 
-(declare-datatype Policy 
-  ((p (principal Int)) 
+(declare-datatype Policy
+  ((p (principal Int))
    (por (left Policy) (right Policy))))
 
 (synth-fun mkPolicy ((d Doc)) Policy
-  ((Start Policy (Q))
-   (Q Policy ((p 0) (p 1) (por Q Q))))
+  ((Q Policy))
```

Comment:
nit: Technically we should have 2 non-terminal symbols, Start and Q here.

Reply:
Addressed.